### ISSUE TITLE
Editing the inlineKeyboard after an Inline request

### ISSUE THREAD

**Issue by neyrodzen** [DinoLeung/TeleDart#263]
When I call the editMessageReplyMarkup method in the onCallbackQuery handler after pressing the keyboard button (InlineKeyboard) there are 2 options: 
1) If the keyboard is sent by a bot, then it is edited and there are no errors
2) If the keyboard is sent after an Inline request (on behalf of the user), then an error comes, although the keyboard is still edited
"Unhandled exception:
TelegramException: Edited message is NOT sent by the bot"
Although the bot is added to group c as an administrator with full rights, and even in a private chat this happens

**Comment by HeySreelal** [comment]
Well, I did a quick test on this and it seems like a bug with the library itself.

According to [documentation](https://core.telegram.org/bots/api#editmessagereplymarkup), the `editMessageReplyMarkup` will return `Message` when the edited message is sent by the bot, otherwise `True` if the message is an inline message sent by the user via the bot.

But, the implementation of the `Telegram.editMessageReplyMarkup` method on this library is to return `Future<Message>` on either case. And to be exact, the method forever throw the exception whenever the API call results in `True` as per the implementation:

https://github.com/DinoLeung/TeleDart/blob/d0b8ec1288f92c5990ad581602e8787f7c6ea580/lib/src/telegram/telegram.dart#L2425-L2428

## TL;DR

The implementation of `Telegram.editMessageReplyMarkup` is to throw the `TelegramException` whenever the bot edits an inline message. In simple words, this is a bug with this library. 🐞

## So what exactly should I do at this point?

As a quick patch, please do a `try-catch` to handle the error on your side, in the mean time, I'll try to add a PR to fix this. Thanks for reporting. 

I'll also look forward for @DinoLeung's thoughts on this since I'm not sure if I should make the method `dynamic` be able to return both `bool` and `Message`.
